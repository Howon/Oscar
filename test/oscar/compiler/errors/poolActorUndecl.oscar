message testNow()


actor Tester() {

  receive = {
    | testNow() => {
        Println("Error");
      }
    | die() => {
      }
  }
}

def main() => unit = {
  pool<Worker> pool1 = spawn pool<Worker>({}, 3);
  message<testNow>() |>> pool1;
  message<die>() |>> pool1;
}
