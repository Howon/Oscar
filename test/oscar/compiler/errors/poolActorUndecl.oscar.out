Error: Cannot create a pool with undeclared actor Worker
